.circleci/config.yml

```diff
@@ -91,12 +99,11 @@ workflows:
   build-test-npm:
     jobs:
```

Comment:
Same with this object here. Seems like each of these items in the `jobs` object needs indented a level? At least in regular YAML that's what I would expect to see. But, I'm not sure if CircleCI requires different formatting. I would expect to see:

```yml
workflows:
  version: 2
  build-test-npm:
    jobs:
      - build
      - build_test:
          requires:
          - build
      - publish_package:
          requires:
            - build_test
          context: iubot
          filters: *filter-only-master
```